hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientMapReduceProxy.java

```diff
@@ -269,25 +261,10 @@ public boolean cancel(boolean mayInterruptIfRunning) {
         }
 
         @Override
-        public boolean isCancelled() {
-            return cancelled;
-        }
-
-        @Override
-        public void setResult(Object result) {
+        protected void setResult(Object result) {
             super.setResult(result);
-            latch.countDown();
```

Comment:
I don't get it... if you delete `@Override protected setResult(x) { super.setResult(x); }`, you should be able to continue using it just the same, unless there's another class in the same package that wants to call `setResult` from the outside.